packages/web-components/@groupby/elements-sayt/src/sayt.ts

```diff
@@ -23,6 +23,8 @@ import {
 } from '@groupby/elements-events';
 // eslint-disable-next-line import/no-extraneous-dependencies, import/no-unresolved
 import { Base } from '@groupby/elements-base';
+// eslint-disable-next-line import/no-extraneous-dependencies, import/no-unresolved
+import { Autocomplete } from '@groupby/elements-autocomplete';
```

Comment:
I would be a little bit wary about this because this may run the autocomplete file, including the `@customElement` decorator. Now that I think about it, I think the reason we used `any` was to get around having to include the other components like this (not exactly a good reason, though).

That said, in this case, because `Autocomplete` is only being used as a type, TypeScript is smart enough to compile it out of the JavaScript bundle so that nothing from `@groupby/elements-autocomplete` is actually run. However, this does mean that we'll have to add `@groupby/elements-autocomplete` as a dev dependency when we get around to doing so.

In short, no action needed, but things to keep in mind.

Reply:
Given that this is being introduced now, shouldn't `@groupby/elements-autocomplete` be added as a dev dependency to this package as part of this ticket?

Reply:
We haven't been adding our packages as dev dependencies. I assumed we were waiting to publish? Wouldn't ``@groupby/elements-base`` also be a dev dependency of sayt?

Reply:
`@groupby/elements-base` would be a real dependency of sayt. We haven't released anything yet, so we can push this off until we've published our packages (otherwise we don't have anything to point to)